templates/liar/templates.yaml

```diff
@@ -0,0 +1,78 @@
+dataset: liar
+templates:
+  331fec6f-bc64-4a81-b895-b1c5d5a61e7e: !Template
+    id: 331fec6f-bc64-4a81-b895-b1c5d5a61e7e
+    jinja: 'What was the party affiliation of the speaker of the following statement?
+
+      ===
+
+      {{statement}} |||
+
+      {{party_affiliation}}'
+    name: Template_5
+    reference: ''
+  770542dc-eeea-464b-9288-1c261030257c: !Template
+    id: 770542dc-eeea-464b-9288-1c261030257c
+    jinja: 'Given the statement below:
+
+      {{statement}}
+
+      ===
+
+      In which of the following categories {{"false", "half-true", "mostly-true",
+      "true", "barely-true", "pants-fire"}} would you label it based on its truthfulness?
+      |||
+
+      {{["false", "half-true", "mostly-true", "true", "barely-true", "pants-fire"][label]}}'
+    name: Template_2
+    reference: ''
+  85cad229-265b-440e-93a0-9264c3209f19: !Template
+    id: 85cad229-265b-440e-93a0-9264c3209f19
+    jinja: 'Given the following statement:
+
+      {{statement}}
+
+      ===
+
+      and the following speaker: {{speaker}}, predict the speaker''s job title. |||
+
+      {{job_title}}'
+    name: Template_3
+    reference: ''
+  c235a60e-5604-4fad-b020-df7f333bdac5: !Template
+    id: c235a60e-5604-4fad-b020-df7f333bdac5
+    jinja: 'Given the following statement:
+
+      {{statement}}
+
+      ===
+
+      To what subject would you categorize it? |||
+
+      {{subject}}'
+    name: Template_1
+    reference: ''
+  d5b1af62-fc94-4031-9e9f-8ea6532926e0: !Template
+    id: d5b1af62-fc94-4031-9e9f-8ea6532926e0
+    jinja: 'Guess the context in which the statement was stated by speaker {{speaker}},
+      who is a {{job_title}}
+
+      ===
+
+      {{statement}} |||
+
+      {{context}}'
```

Comment:
Same comment about providing choices for the context, if they are available.

Reply:
There are no options for providing choices for the context here as well. So should we keep the prompt the way it is?

Reply:
Sounds good.